client/lib/tos-acceptance-tracking/index.js

```diff
@@ -0,0 +1,19 @@
+import { getLocaleSlug } from 'calypso/lib/i18n-utils';
+
+export default function () {
```

Comment:
Minor, but it may help future developers to find this function if it has a name instead of being anonymous. Since it's being imported everywhere as `getToSAcceptancePayload` that would probably be a good name to give it here too.

Reply:
The function is now named. Thanks for pointing this out!